Octokit.Tests.Integration/Clients/RepositoryCollaboratorClientTests.cs

```diff
@@ -136,4 +136,26 @@ public class TheIsCollaboratorMethod
             }
         }
     }
+
+    public class TheInviteMethod
+    {
+        [IntegrationTest]
+        public async Task CanInviteNewCollaborator()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoName = Helper.MakeNameWithTimestamp("public-repo");
+
+            using (var context = await github.CreateRepositoryContext(new NewRepository(repoName)))
+            {
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest();
+
+                // invite a collaborator
+                var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, "maddin2016", permission);
```

Comment:
youre right. **When** there is a pending invite. Ok. Then i use my `lrz-hal` account.